beacon-chain/sync/rpc_light_client.go

```diff
@@ -26,7 +27,7 @@ func (s *Service) lightClientBootstrapRPCHandler(ctx context.Context, msg interf
 
 	SetRPCStreamDeadlines(stream)
 	if err := s.rateLimiter.validateRequest(stream, 1); err != nil {
-		logger.WithError(err).Error("s.rateLimiter.validateRequest")
+		logger.WithError(err).Error("Canno validate request")
```

Comment:
typo?

Reply:
Fixed in a9a678e4c74f5e08f452e38e9e2d06550e95698d.